### ISSUE TITLE
maxImageCacheCount does not work on iOS devices

### ISSUE THREAD

**Issue by wizeobara** [openseadragon/openseadragon#2176]
OpenSeaDragon version 2.3.1

```
Total canvas memory use exceeds the maximum limit (224 MB).
TypeError: null is not an object (evaluating 'this._renderedContext.drawImage')
```

Above error is shown when running on XCode Simulator iPad 9th generation iOS 15.2.

I changed maxImageCacheCount to 60 from default 200 to prevent canvas memory from exceeding limit.
However, I still see over 128 canvas image in Graphics Tab in Web Inspector.
 
Is it caused by same problem as link below?
https://pqina.nl/blog/total-canvas-memory-use-exceeds-the-maximum-limit/

**Comment by iangilman** [comment]
Yes, that's a likely culprit! Releasing our canvases like they suggest in that post sounds like a good plan.

Would you be up for working up a fix? If so, I'm happy to help point you in the right direction. If not, hopefully someone will take this on.
